src/proposals/20250109/SparkBase_20250109.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { SparkPayloadBase, Base } from "src/SparkPayloadBase.sol";
+
+import { Ethereum } from "spark-address-registry/Ethereum.sol";
+
+import { CCTPForwarder } from "xchain-helpers/forwarders/CCTPForwarder.sol";
+
+import { IMetaMorpho }  from "metamorpho/interfaces/IMetaMorpho.sol";
+import { MarketParams } from "morpho-blue/src/interfaces/IMorpho.sol";
+
+import { ControllerInstance }    from "spark-alm-controller/deploy/ControllerInstance.sol";
+import { ForeignControllerInit } from "spark-alm-controller/deploy/ForeignControllerInit.sol";
+
+/**
+ * @title  Jan 9, 2025 Spark Base Proposal
+ * @notice Onboard Aave aUSDC and Morpho USDC Vault
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/27-dec-2024-proposed-changes-to-spark-for-upcoming-spell/25760
+ * Vote:   TODO
```

Comment:
https://vote.makerdao.com/polling/QmQarR2U

Reply:
will there be a poll for the update to the ALM controller?

Reply:
It's implied by the addition of Ethena, Aave and Morpho because otherwise how would you add support.

Reply:
added polls